internal/database/mysql.go

```diff
@@ -559,11 +574,28 @@ func (db *MySQLDatabase) GetAccounts(ctx context.Context, opts GetAccountsOption
 	query += ` ORDER BY username`
 
 	// Fetch list account
-	accounts := []model.Account{}
-	err := db.SelectContext(ctx, &accounts, query, args...)
-	if err != nil && err != sql.ErrNoRows {
+	rows, err := db.Queryx(query, args...)
```

Comment:
You should be able to use the same `SelectContext` here since `model.Account` implements [`driver.Scanner`](https://pkg.go.dev/database/sql#Scanner). Did you encounter any issues?

My guess is the same for all database engines.